### PR TITLE
#3101 Redesign ETO documents

### PR DESCRIPTION
**Pull request by drakkein**

### CODE REVIEW

File: packages/web/app/modules/eto-documents/sagas.ts [hunk 1]
```diff
@@ -198,6 +198,19 @@ function* uploadEtoFileEffect(
   notificationCenter.info(createMessage(EtoDocumentsMessage.ETO_DOCUMENTS_FILE_UPLOADED));
 }
 
+function* removeEtoFileEffect(
+  { apiEtoFileService, notificationCenter }: TGlobalDependencies,
+  documentType: EEtoDocumentType,
+): Iterator<any> {
+  const matchingDocument = yield getDocumentOfType(documentType);
```
Comment (Moejoe90) [inline packages/web/app/modules/eto-documents/sagas.ts]: and what happens if there is no `matchingDocument` 
We sholdn't show 
```
  notificationCenter.info(createMessage(EtoDocumentsMessage.ETO_DOCUMENTS_FILE_REMOVED));
``` 
if matchingDocument wasn't there 

also report this case to sentry becuase something is really wrong

File: packages/web/app/components/shared/Document.tsx [hunk 2]
```diff
@@ -69,106 +58,74 @@ export const Document: React.FunctionComponent<IDocumentProps> = ({
...
+      {extension}
+    </p>
+    <div className={styles.buttons}>
+      <ButtonIcon
+        data-test-id="documents-download-document"
+        className={styles.buttonIcon}
+        onClick={downloadAction}
+        type="button"
+        svgIcon={download}
+        alt={<FormattedMessage id="button-icon.down" />}
```
[the remark this answers is not in the document]
Comment (drakkein) [reply packages/web/app/components/shared/Document.tsx]: It is hidden behind loading screen, so it should not be possible for user to click on it, anyway I will disable it for better security

File: packages/web/app/modules/eto-documents/sagas.ts [hunk 3]
```diff
@@ -226,12 +239,43 @@ function* uploadEtoFile(
       const eto: TEtoWithCompanyAndContract = yield nonNullable(select(selectIssuerEto));
 
       const uploadResult = Object.values(eto.documents).find(d => d.documentType === documentType)!;
-      yield put(actions.etoFlow.signInvestmentAgreement(eto, uploadResult.ipfsHash));
+
+      // If user does not sign transaction uploadResult is undefined
```
[the remark this answers is not in the document]
Comment (drakkein) [reply packages/web/app/modules/eto-documents/sagas.ts]: Basically if user close permission check modal/metamask signer during upload.

**Comment by drakkein** [comment]
Fixed issue with uploading PDF's.
Fixed styling for templates box.

Downloading is related to CORS, should work normally on '.io'. Codechecks has some headers missing.

**Pull request merged**